### ISSUE TITLE
execute_mdx_dataframe encoding error 

### ISSUE THREAD

**Issue by scotthoopes** [cubewise-code/tm1py#903]
**Describe the bug**

When using the new 1.11 version of TM1Py I implemented the new "use_blob" argument for the execute_mdx_dataframe method. 

When running this code retrieving data from TM1 I am getting the following error: 

 'utf-8' codec can't decode byte 0x96 in position 77793: invalid start byte. 

**To Reproduce**

Using TM1Py 1.11 use the "use_blob" argument as seen below:

tm1.cells.execute_mdx_dataframe(mdx = mdxstr, use_blob=True)

**Expected behavior**
Expecting data to be returned in a dataframe with the correct encoding. 

**Version**
TM1py 1.11
TM1 Server Version: PA Cloud

**Comment by MariusWirtz** [comment]
Thanks for raising the issue. 

Please upgrade to https://github.com/cubewise-code/tm1py/pull/904 and verify that it resolves the issue

```
pip uninstall tm1py
pip install https://github.com/cubewise-code/tm1py/archive/refs/heads/issue/execute-mdx-csv-encoding.zip
```

**Comment by MariusWirtz** [comment]
> @MariusWirtz I ran my script over again with no errors!!
> 
> Thanks!
> 
> As a side note it also fixed some errors I was having when we moved to IBM Cloud.
> 
> Also this new version is lightning fast. This used to take about 35 minutes for 24MM this time it took 12 minutes to run! Very excited.

That is actually a very interesting finding! When I tested the performance of `execute_mdx_dataframe` with `use_blob=True` against the predecessor function in single-threaded execution I found "only" a 20% performance increase. 

But based on your findings multi-threaded `execute_mdx_dataframe` with `use_blob=True` performs 3x better than multi-threaded `execute_mdx_dataframe` with `use_blob=False`. Good incentive to implement https://github.com/cubewise-code/tm1py/issues/887 for the next release :)
